experimental/LieAlgebras/test/LieAlgebraModule-test.jl

```diff
@@ -1122,4 +1122,315 @@
       )
     end
   end
+  @testset "demazure_character" begin
+    function demazure_character_trivial_tests(R::RootSystem, w::WeightLatticeElem)
+      x = longest_element(weyl_group(R))
+      result = demazure_character(w, x)
+      @test result == character(R, w)
+      W = weyl_group(R)
+      result = demazure_character(w, one(W))
+      dict = Dict(w => 1)
+      dict = Dict(Int.(Oscar._vec(coefficients(w_))) => m for (w_, m) in dict)
+      @test result == dict
+    end
+    @testset "Type A" begin
```

Comment:
```suggestion
    # The "correct" solution in the below tests has been computed using Magma V2.28-7.
    # Note that Magma considers right actions of Weyl groups on weights,
    # so to reproduce one needs to reverse all Weyl words when copying a test case to Magma.
    
    @testset "Type A" begin
```
Add a comment so that future generations have a chance to understand what we did here.